setup.py

```diff
@@ -119,6 +120,12 @@ def __str__(self):
     odbclib = "odbc"
 
 
+def is_cxx11_abi():
+    cxx11_abi = subprocess.getoutput("echo '#include <string>' | g++ -x c++ -E -dM - | fgrep _GLIBCXX_USE_CXX11_ABI | \
```

Comment:
The `g++` executable should be exchanged with the compiler detected by the build system. It isn't always called `g++` but could be sometimes `x86-linux-g++` or similar.

Reply:
Would changing it to gcc solve the problem?

Reply:
No, there is some variable you can get from distutils or setuptools that tell you the current CXX compiler executable.

Reply:
Thanks for the tip, did some digging and found a solution, please review